app/controllers/root_controller.rb

```diff
@@ -0,0 +1,3 @@
+class RootController < ApplicationController
+  def index; end
+end
```

Comment:
[OPTIONAL]
- [ ] Since we are not using the root controller it would be nice to remove the `root_controller.rb` file to follow the **YAGNI** principle

Reply:
noted